client/app/css/less/utils/variables.less

```diff
@@ -208,3 +208,9 @@
 // -------------------------
 
 @defaultPopoverFont      : @grey6;
+
+//
+// droupdown menu
+// -------------------------
+
+@oui-dropdown-menu-zindex: 1000;
```

Comment:
Is there a rule to set this only for mobile devices ?

Reply:
It can be done using a media query based on screen size. But it's not required as there is no impact on large screens. I have tested it.

Reply:
Ok, but be careful if later you want to apply a zindex different for larger screens, you could be obliged to set media rules.

Reply:
I agree with @JeremyDec, better safe than sorry 😉